macOS/DuckDuckGo/Menus/MainMenu.swift

```diff
@@ -783,6 +783,7 @@ final class MainMenu: NSMenu {
 
             NSMenuItem(title: "Logging").submenu(setupLoggingMenu())
             NSMenuItem(title: "AI Chat").submenu(AIChatDebugMenu())
+            NSMenuItem(title: "Updates").submenu(UpdatesDebugMenu())
```

Comment:
Use this menu for testing - no need for multiple builds, if you expire the current build you'll be forcing a new update, which can be useful for simulating new builds becoming available.